config/initializers/black_light_silence.rb

```diff
@@ -0,0 +1,8 @@
+# Avalon version five writes a lot of deprecation warnings to
+# stderr (which goes to the apache error log). These warnings
+# are fixed in Avalon version six. 
+if Avalon::VERSION.to_i < 6
+  if [:production, :staging, :test].include?(Rails.env.to_sym)
```

Comment:
In this case, this is the "test" vs "staging" issue. I would actually like to keep this in for "test" (as in the test suite), but the UNIX team is currently using the "test" environment for staging. I am working on the playbook to rename "test" to "staging", but I'm not there yet, so eventually the behavior will be:

1) Do warnings for dev and test
2) Don't do warnings for production and staging.